contracts/interchain-token-service/src/contract/execute/mod.rs

```diff
@@ -227,9 +321,22 @@ fn send_to_destination(
     storage: &dyn Storage,
     querier: QuerierWrapper,
     destination_chain: &ChainNameRaw,
-    destination_address: Address,
     payload: HexBinary,
 ) -> Result<Response, Error> {
+    if *destination_chain == axelar_chain_name(storage, querier)? {
+        // messages sent to axelar are handled in a special way
```

Comment:
What's the special way right now?

Reply:
they don't send anything to another destination chain after the hub is done with the message

Reply:
yeah it's what I describe below. Maybe I can just remove this specific line though